src/components/Calculator.js

```diff
@@ -1,31 +1,52 @@
 import '../App.css';
 import React from 'react';
+import calculate from '../logic/calculate';
 
 class Calc extends React.PureComponent {
+  constructor() {
+    super();
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
+
+  clickHandler = (e) => {
+    const button = e.target.innerHTML;
+    const result = calculate(this.state, button);
+    this.setState(result);
+  };
+
```

Comment:
- [ ] In Reading the Project requirements, It says that:
- Use `state` and `props`: 
- `Props`  need to get passed to child components of the Calculate.js component.
- Please implement `props` and fulfill the project requirements